app/templates/results.html

```diff
@@ -35,19 +35,28 @@ <h4>Timespans covered:</h4>
         </p>
       {% endif %}
       {% if nojs is not defined %}
-        <div class="abstract abstract--truncated" role="region" aria-expanded="false">
+        <div class="result_field result--truncated" role="region" aria-expanded="false">
           {{ result.abstract|safe|truncate(500) }}
+           
+          
+
         </div>
       {% endif %}
-      <div class="abstract abstract--full" role="region" {% if nojs is not defined %}aria-hidden="true"{% endif %}>
+      <div class="result_field result--full" role="region" {% if nojs is not defined %}aria-hidden="true"{% endif %}>
         {{ result.abstract|safe}}
         {% if result.license %}
               <div class="license">
                 <h4>License:</h4>
                   {{ result.license }}
+
+
               </div>
         {% endif %}
+        
       </div>
+      <button class="show_more_button" id ="show_more">Show more</button>
```

Comment:
Please take the ids out of these - ids have to be unique *per page*, and because they show up in each search result, duplicate ids will make some browsers do weird stuff.